README.md

```diff
@@ -0,0 +1,88 @@
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+![](https://github.com/FAIRmat-NFDI/pynxtools-xrd/actions/workflows/pytest.yml/badge.svg)
+![](https://github.com/FAIRmat-NFDI/pynxtools-xrd/actions/workflows/pylint.yml/badge.svg)
+![](https://github.com/FAIRmat-NFDI/pynxtools-xrd/actions/workflows/publish.yml/badge.svg)
+![](https://img.shields.io/pypi/pyversions/pynxtools-xrd)
+![](https://img.shields.io/pypi/l/pynxtools-xrd)
+![](https://img.shields.io/pypi/v/pynxtools-xrd)
+![](https://coveralls.io/repos/github/FAIRmat-NFDI/pynxtools_xrd/badge.svg?branch=master)
+
+# XRD Reader
+With the XRD reader, data from X-ray diffraction experiment can be read and written into a NeXus file (h5 type file with extension .nxs) according to NXxrd_pan application definition in [NeXus](https://github.com/FAIRmat-NFDI/nexus_definitions). There are a few different methods of measuring XRD: 1. θ:2θ instruments (e.g. Rigaku H3R), and 2. θ:θ instrument (e.g. PANalytical X’Pert Pro). The goal with this reader is to support both of these methods.
+
+**NOTE: This reader is still under development. As of now, the reader can only handle files with the extension `.xrdml` , obtained with PANalytical X’Pert Pro version 1.5 (method 2 described above). Currently we are wtoking to include more file types and file versions.**
+
+# Installation
+
+It is recommended to use python 3.11 with a dedicated virtual environment for this package.
```

Comment:
Do you think it is 3.11 is the best one or 3.9 regarding the compatibility with nomad.
https://gitlab.mpcdf.mpg.de/nomad-lab/nomad-FAIR/-/blob/develop/requirements.txt?ref_type=heads